components/desktop/gnome-icon-theme-symbolic/manifests/sample-manifest.p5m

```diff
@@ -10,10 +10,11 @@
 #
 
 #
-# Copyright 2016 <contributor>
+# Copyright 2024 <contributor>
 #
 
 set name=pkg.fmri value=pkg:/$(COMPONENT_FMRI)@$(IPS_COMPONENT_VERSION),$(BUILD_VERSION)
+set name=pkg.human-version value=$(HUMAN_VERSION)
```

Comment:
I assume the real manifest needs an update too.

Reply:
done - and thanks for the heads-up